<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MoveMoneyFromCardToCard</class>
 <widget class="QDialog" name="MoveMoneyFromCardToCard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="cardNumber1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Отправитель</string>
   </property>
   <property name="maxLength">
    <number>16</number>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>341</width>
     <height>471</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{background: #F9F9F9;
border-radius: 20px;}
</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="createButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background: #81C4FF; border-radius: 15px; color: #e6f6ff; font-size: 10pt; font-weight: bold;} 
QPushButton:hover{background: #e6f6ff; border-radius: 15px; color: #81C4FF; font-size: 10pt; font-weight: bold;}</string>
    </property>
    <property name="text">
     <string>Перевести</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sum">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>200</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>170</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Сумма&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pin">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #d9ebff;
border-radius: 10px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>4</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>160</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;PIN-код&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>-30</y>
      <width>241</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/images/images/credit-card.png);
background: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>241</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/images/images/credit-card.png);
background: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="cardNumber_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>320</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Получатель</string>
    </property>
    <property name="maxLength">
     <number>16</number>
    </property>
   </widget>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>pushButton_5</zorder>
   <zorder>createButton</zorder>
   <zorder>cardNumber_2</zorder>
   <zorder>pin</zorder>
   <zorder>sum</zorder>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>430</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover{background: #ffafa5; border-radius: 5px; border: 0.2px solid #d7eba9; color: #ffffff; font-size: 9pt; font-weight: bold;}																
QPushButton{background: #d9d9d9; border-radius: 5px; color: #262626; font-size: 9pt; font-weight: bold;}</string>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>cardNumber1</zorder>
  <zorder>cancel</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
